utils/wasm-gen/src/generator/syscalls/invocator.rs

```diff
@@ -169,21 +171,23 @@ impl<'a, 'b> SysCallsInvocator<'a, 'b> {
             self.unstructured.len()
         );
 
+        let mut invoke_instructions = vec![];
```

Comment:
`with_capacity`

Reply:
I'll need to iterate over `sys_calls_imports` to achieve this. And also this `Vec` contains `Vec`s, which have the size of around 16 bytes

Reply:
what about calling inserting sys-calls invocation instructions only for one sys-call per iteration, but not to collect all instructions?